modules/electrophysiology/src/main/resources/schemas/neurosciencegraph/electrophysiology/stimulusexperiment/v0.1.1.json

```diff
@@ -0,0 +1,73 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/electrophysiology/stimulusexperiment/v0.1.1/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.3",
+    "{{base}}/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.1"
+  ],
+  "shapes": [
+    {
+      "@id": "this:StimulusExperimentShape",
+      "@type": "sh:NodeShape",
+      "label": "Stimulus experiment shape definition",
+      "targetClass": "nsg:StimulusExperiment",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.3/shapes/ActivityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:stimulus",
+              "name": "Stimulus",
+              "description": "The shape of the stimulus",
+              "node": "this:StimulusShape",
+              "minCount": 1,
+              "maxCount": 1
+            },
+            {
+              "path": "prov:used",
+              "description": "The cell recorded from",
+              "class": {
```

Comment:
I suggest to use nsg:Cell as a type of the used entity.

Reply:
I'm not sure that is specific enough. It is important that the object used in the experiment be a cell-that-has-been-recorded-from. 

I had in mind to define "nsg:RecordedCell" as a parent of "nsg:PatchedCell" and "nsg:IntraCellularSharpElectrodeRecordedCell", but I didn't do that yet because I wanted to keep the changes minimal so as to get them deployed as soon as possible.

Reply:
Thank you for the clarification. I'm all for using nsg:RecordedCell then. I think it makes sense.